File: app/src/main/java/com/pnuema/android/bible/ui/viewholders/CopyrightViewHolder.java
package com.pnuema.android.bible.ui.viewholders;

import android.view.LayoutInflater;
import android.view.ViewGroup;
import android.widget.TextView;

import com.pnuema.android.bible.R;
import com.pnuema.android.bible.statics.HtmlUtils;

import androidx.annotation.NonNull;
import androidx.recyclerview.widget.RecyclerView;

public class CopyrightViewHolder extends RecyclerView.ViewHolder {
    private TextView copyrightText;

    public CopyrightViewHolder(@NonNull final ViewGroup parent) {
        super(LayoutInflater.from(parent.getContext()).inflate(getLayout(), parent, false));

        copyrightText = itemView.findViewById(R.id.copyrightText);
    }

    public void bind(final String text) {
        copyrightText.setText(HtmlUtils.fromHtml(text));
    }

    public static int getType() {
        return getLayout();
    }

    private static int getLayout(){
        return R.layout.listitem_copyright;
    }
}

File: app/src/main/java/com/pnuema/android/bible/ui/viewholders/CopyrightViewHolder.kt
package com.pnuema.android.bible.ui.viewholders

import android.view.LayoutInflater
import android.view.ViewGroup
import android.widget.TextView
import androidx.recyclerview.widget.RecyclerView
import com.pnuema.android.bible.R
import com.pnuema.android.bible.statics.HtmlUtils

class CopyrightViewHolder(parent: ViewGroup) : RecyclerView.ViewHolder(LayoutInflater.from(parent.context).inflate(layout, parent, false)) {
    private val copyrightText: TextView = itemView.findViewById(R.id.copyrightText)

    companion object {

        val type: Int
            get() = layout

        private val layout: Int
            get() = R.layout.listitem_copyright
    }

    fun bind(text: String) {
        copyrightText.text = HtmlUtils.fromHtml(text)
    }
}
